.github/actions/changed-packages/action.yml

```diff
@@ -1,9 +1,9 @@
 # Example usage in `./workflows/reusable-e2e.yml`
 name: Check Changed Packages
-description: 'Checks whether any files in the specified packages were changed in the current commit'
+description: 'Checks whether any files in the specified packages were changed in the current PR'
 inputs:
-  commit:
-    description: 'Commit to check files against'
+  pr-branch:
```

Comment:
Instead of comparing `PR_BRANCH` to `main`, can we explicitly have two inputs `base-commit` and `head-commit` and compare the two?

Think that'd be nicer if we intend to use action in feature branches for e2e purposes. Base ref is available on `github.event.pull_request.base.sha` iirc.